docs/ReleaseNotes.md

```diff
@@ -1,4 +1,6 @@
 # Release notes
+## v1.2
+TBD 
```

Comment:
Under `Features` add the following:
i) Added Health check and APIs to fetch clusters and admiral generated end points.
ii) Added support/testing for istio versions 1.8.x, 1.9.x and 1.10.x with smart DNS proxying

Reply:
Do I need to link issue like in previous features? I'm having a little trouble finding the API related issue. And just want to double check for second feature, is the issue this one:  https://github.com/istio-ecosystem/admiral/issues/164? @aattuluri